core/src/epicli/data/common/ansible/playbooks/roles/filebeat/defaults/main.yml

```diff
@@ -1,3 +1,5 @@
 ---
 filebeat_helm_chart_file_name: filebeat-7.9.2.tgz
 filebeat_version: "7.9.2"
+# Use custom namespace for logging charts such as filebeat in case of k8s as cloud service.
+logging_chart_namespace: epi-logging
```

Comment:
Shouldn't it be configurable?

Reply:
We've decided to keep it in defaults for now. After this refactor #1756 it will be easier to properly implement upgrades for this case.